inputfiles/overridingTypes.jsonc

```diff
@@ -286,6 +286,138 @@
+                        "ariaRequired": {
+                            "overrideType": "\"true\" | \"false\" | (string & {})"
+                        },
+                        // "ariaRoleDescription": {
+                        //     "overrideType": ""
+                        // },
+                        "ariaRowCount": {
+                            "overrideType": "`${bigint}` | \"\""
+                        },
+                        "ariaRowIndex": {
+                            "overrideType": "`${bigint}` | \"\""
+                        },
+                        // "ariaRowIndexText": {
+                        //     "overrideType": ""
+                        // },
+                        "ariaRowSpan": {
+                            "overrideType": "`${bigint}` | \"\""
+                        },
+                        "ariaSelected": {
+                            "overrideType": "\"true\" | \"false\" | \"undefined\" | (string & {})"
+                        },
+                        "ariaSetSize": {
+                            "overrideType": "`${bigint}` | \"\""
+                        },
+                        "ariaSort": {
+                            "overrideType": "\"ascending\" | \"descending\" | \"none\" | \"other\" | (string & {})"
+                        },
+                        "ariaValueMax": {
+                            "overrideType": "`${number}` | \"\""
+                        },
+                        "ariaValueMin": {
+                            "overrideType": "`${number}` | \"\""
+                        },
+                        "ariaValueNow": {
+                            "overrideType": "`${number}` | \"\""
+                        },
+                        // "ariaValueText": {
+                        //     "overrideType": ""
+                        // }
+                        "role": {
+                            "overrideType": "\"toolbar\" | \"tooltip\" | \"feed\" | \"math\" | \"presentation\" | \"none\" | \"note\" | \"application\" | \"article\" | \"cell\" | \"columnheader\" | \"definition\" | \"directory\" | \"document\" | \"figure\" | \"group\" | \"heading\" | \"img\" | \"list\" | \"listitem\" | \"meter\" | \"row\" | \"rowgroup\" | \"rowheader\" | \"separator\" | \"table\" | \"term\" | \"associationlist\" | \"associationlistitemkey\" | \"associationlistitemvalue\" | \"blockquote\" | \"caption\" | \"code\" | \"deletion\" | \"emphasis\" | \"insertion\" | \"paragraph\" | \"strong\" | \"subscript\" | \"superscript\" | \"time\" | \"scrollbar\" | \"searchbox\" | \"separator\" | \"slider\" | \"spinbutton\" | \"switch\" | \"tab\" | \"tabpanel\" | \"treeitem\" | \"combobox\" | \"menu\" | \"menubar\" | \"tablist\" | \"tree\" | \"treegrid\" | \"banner\" | \"complementary\" | \"contentinfo\" | \"form\" | \"main\" | \"navigation\" | \"region\" | \"search\" | \"alert\" | \"log\" | \"marquee\" | \"status\" | \"timer\" | \"alertdialog\" | \"dialog\" | (string & {})"
```

Comment:
Where did you get this list? Have you checked everything here has multiple implementation support?